pkg/fdbstatus/status_checks_test.go

```diff
@@ -1121,18 +1121,147 @@ var _ = Describe("status_checks", func() {
 			})
 		})
 
-		Context("coordinator fault domain checks", func() {
-			It("no coordinator related information in status", func() {
+		When("doing the coordinator fault domain checks", func() {
+			BeforeEach(func() {
 				status = &fdbv1beta2.FoundationDBStatus{
+					Client: fdbv1beta2.FoundationDBStatusLocalClientInfo{
+						Coordinators: fdbv1beta2.FoundationDBStatusCoordinatorInfo{
+							QuorumReachable: true,
```

Comment:
Can we export this to a variable and use it in all the test cases as that code is repeated multiple times in this file ?

Reply:
We change this value exactly once, so I'm not sure about the value of moving it into a variable.